src/sentry/sentry_metrics/consumers/indexer/batch.py

```diff
@@ -39,6 +39,7 @@
 MAX_TAG_VALUE_LENGTH = 200
 
 ACCEPTED_METRIC_TYPES = {"s", "c", "d"}  # set, counter, distribution
+MRI_RE_PATTERN = re.compile("^([c|s|d|g|e]):([a-zA-Z0-9_]+)/.*$")
```

Comment:
Where does this come this from? Does it match the regex we have in Relay or something?

Reply:
That's correct, it picks up the name field from the MRI
https://getsentry.github.io/relay/relay_metrics/struct.Metric.html#structfield.name